src/app.js

```diff
@@ -18,9 +23,38 @@ export default class Flashcard extends Component{
     this.setState({cards: flashcards})
   }
 
+  renderView() {
+    const { path } = this.state.view
+    const { cards } = this.state
+    switch (path) {
+      case 'new':
```

Comment:
Please review your view rendering logic in this method. `#new` should always show a form to create a card. `#cards` should always show **a)** a notice that the list is empty or **b)** the list of cards.

Try conditionally rendering your "no cards" message from inside the `Deck`. That way the user can always go to the `Deck` screen, but it might tell them to create a card.

Right now, if the hash is something random `#foo`, then the user will be told they have no cards (which is not true).